sickbeard/providers/torrentz.py

```diff
@@ -90,7 +90,7 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                             if item.category and 'tv' not in item.category.get_text(strip=True):
                                 continue
 
-                            title = item.title.text.rsplit(' ', 1)[0].replace(' ', '.')
+                            title = item.title.text.rsplit(' ', 1)[0].replace('x264 ', 'x264-').replace('h264 ', 'h264-').replace('h 264 ', 'h264-').replace('xvid ', 'xvid-').replace(' ', '.')
```

Comment:
Regex is heavier than a single replace, but after repeated replaces and the fact that replace is case sensitive, and regex's would match better, its probably a better choice.  

Is your second point related to the regex recommendation?